tests/functional/d/docstrings_fixme.py

```diff
@@ -0,0 +1,27 @@
+# pylint: disable=unnecessary-pass, consider-using-f-string
+
+"""
+Test fixme in docstrings.
+"""
+
+def func1(): # [fixme-in-docstring]
+    """
+    TODO: Implement
+    """
+
+# fixme-in-docstring
+def func2(): # [fixme-in-docstring]
+    """
+    FIXME: Implement
+    """
+
+# fixme-in-docstring
+def func3(): # [fixme-in-docstring]
+    """
+    XXX: Implement
```

Comment:
Could you add a test where the `FIXME` is not the first word of the line? And also one where it is not surrounded by spaces?